bin/run_docker_test

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
```

Comment:
Is this file necessary? What does it do that the existing `bin/run_integration_tests` does not? I would rather add a feature or two to that script than add a bunch of cruft we don't need and don't understand to the repo.